.changeset/wise-rules-march.md

```diff
@@ -0,0 +1,5 @@
+---
+"miniflare": minor
+---
+
+feat: add Dev Registry support
```

Comment:
I think the changeset could use some extra context 🙂

Reply:
Updated in [bade61f](https://github.com/cloudflare/workers-sdk/pull/9313/commits/bade61fd44b6832bc15c94294a20be2600a7d88d)